internal/pkg/util/fs/squashfs/squashfs.go

```diff
@@ -60,6 +62,16 @@ func GetProcs() (uint, error) {
 	// proc is either "" or the string value in the conf file
 	proc := c.MksquashfsProcs
 
+	// let user override via ENV
+	procEnv := os.Getenv("SINGULARITY_MKSQUASHFS_PROCS")
```

Comment:
@soichih as for naming ... perhaps consider that the naming would be cleaner and closer to a canonical form with the suffix "_PROCESSORS" instead of "_PROCS" ... the reason being is that the mksquashfs command uses "-processors N" as its command line option